### DISCUSSION TITLE
User Personas

### DISCUSSION THREAD

**Discussion by alasaidi** [HYF-Class22/agile-development-group2-hyf-redesign#2] [Ideas]
# User Personas

---

### Bio:
- Age: 27
- Educational Background: Bachelor's degree in Computer Science
- Coding Experience: Limited, basic understanding of HTML and CSS

### Needs/Goals:
- Acquire practical coding skills to transition into a career as a web developer.
- Access affordable and quality coding courses to enhance knowledge.
- Receive mentorship and support to navigate the challenges of learning to code.
- Build a portfolio showcasing projects for job applications.
- Connect with a community of fellow learners and industry professionals.

### Frustrations:
- Limited time due to a full-time job, seeking a flexible learning schedule.
- Uncertainty about where to start and progress in the coding journey.
- Difficulty finding affordable and reputable online coding resources.
- Feeling isolated in the learning process, lacking guidance and mentorship.
- Concerns about the practical application of coding skills in real-world projects.

### Scenarios:

- **Starting the Learning Journey:** Alex is new to coding and wants to start with the basics. He needs a user-friendly platform with beginner-friendly courses that provide a structured learning path.

- **Balancing Learning and Work:** Alex needs a platform that offers flexible schedules and on-demand learning materials to accommodate his busy schedule.

- **Engaging with a Community:** Feeling isolated in his learning journey, Alex wants to connect with other learners and industry professionals. He looks for forums, discussion boards, or networking events on the platform.

---

### Bio:
- Age: 27
- Educational Background: Bachelor's degree in Marketing
- Coding Experience: No background in web development or IT

### Needs/Goals:
- **Career Transition:** Emily aims to acquire practical coding skills to transition into a career as a web developer.
- **Access to Beginner-friendly Courses:** She is looking for affordable and beginner-friendly coding courses to build a foundation.
- **Mentorship and Support:** Emily seeks mentorship and support to navigate the challenges of learning to code without any prior background.

### Frustrations:
- **Limited Time:** Due to a full-time job in marketing, Emily faces time constraints and requires a flexible learning schedule.
- **Uncertainty in Learning Path:** She is unsure about where to start and how to progress in her coding journey with no prior background.
- **Affordable and Reputable Resources:** Emily finds it challenging to locate affordable and reputable online coding resources suitable for beginners.
- **Practical Application Concerns:** Emily is concerned about the practical application of coding skills in real-world projects without a tech background.

### Scenarios:
- **Starting the Learning Journey:** Emily, being entirely new to coding, wants to start with the basics. She needs a user-friendly platform with beginner-friendly courses that provide a structured learning path.
- **Balancing Learning and Work:** Emily, with a full-time job in marketing, needs a platform that offers flexible schedules and on-demand learning materials to accommodate her busy schedule.

Adapting the content to Emily's background ensures that the Hack Your Future website addresses the unique challenges and needs of someone with no prior web development or IT experience.

**Comment by Prince4ademola** [comment]
This is good
